apps/dashboard/pkg/apis/dashboard/v0alpha1/search.go

```diff
@@ -61,18 +61,20 @@ type DashboardHit struct {
 	Resource string `json:"resource"` // dashboards | folders
 	// The k8s "name" (eg, grafana UID)
 	Name string `json:"name"`
-	// The display nam
+	// The display name
 	Title string `json:"title"`
+	// Dashboard description
+	Description string `json:"description,omitempty"`
 	// Filter tags
 	Tags []string `json:"tags,omitempty"`
 	// The k8s name (eg, grafana UID) for the parent folder
 	Folder string `json:"folder,omitempty"`
 	// Stick untyped extra fields in this object (including the sort value)
-	Field *common.Unstructured `json:"field,omitempty"`
+	Field *common.Unstructured `json:"field,omitzero,omitempty"`
```

Comment:
^^^ this may have consequences that I have not checked

it previously a null field for most results
```
{
   field: null
}
```